email/200903.html

```diff
@@ -1,105 +0,0 @@
-<html>
```

Comment:
I put all these files in an Archive folder in our Google Drive

Reply:
Only concern is that they might be actual links in existing emails (like "click here if this doesn't render in your email reader" type stuff).  The images might be important too.

Of course, they're 10+ years old, so... not super relevant.